wine_worker_container/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:20.04
```

Comment:
Why are we using ubuntu instead of debian? I think we can make the image lighter that way. Or even go for alpine.

Reply:
Trochę dziwne, ale obraz na debian 12 light waży 3.5gb a na ubuntu 2.5gb

Reply:
Udalo się zmniejszyć do 1.86gb